deployment/src/kubernetes/README.md

```diff
@@ -280,7 +280,7 @@ helm uninstall ocelot \
 
 ## Backups
 
-You can and should do [backups](/deployment/kubernetes/Backup.md) with Kubernetes for sure.
+You can and should do [backups](./Backup.md) with Kubernetes for sure.
```

Comment:
Absolut path?

Reply:
In case linked markdown files are in the same directory, relative paths work for both Github and Vuepress.